### DISCUSSION TITLE
Webhooks issue since v2 retirement

### DISCUSSION THREAD

**Discussion by assaflei** [etsy/open-api#945] [Q&A]
We recieve error messages when pubsubhubbub webhooks are invoked
The message we get found below

This is sent as a query string to our callback url so not really clear what action is represented here
It also doesn't happen a lot, we are still getting valid webhooks updates as well

Any advice?

```
{"hub.mode":"denied","hub.reason":"{"error":"API v2 deprecated.",  
"message":"The API version 2 you are trying to access has been deprecated. Please upgrade to the latest version (v3) as soon as possible to resume service. For documentation on how to use API v3, please visit https://developers.etsy.com/documentation/migration/index."}","hub.topic":"http://www.etsy.com/api/push/shops/_shop_name_/listings/latest.atom"}
```

**Comment by Gareth064** [comment]
Yeah, as of 3rd April that service, which was part of the v2 API offering, is now shutdown. The v3 API doesn't offer a replacement I'm afraid.
